src/sql/run.py

```diff
@@ -179,6 +184,18 @@ def __getitem__(self, key):
                 raise KeyError('%d results for "%s"' % (len(result), key))
             return result[0]
 
+    def __getattribute__(self, attr):
+        "Raises AttributeError when invalid operation is performed."
+        try:
+            return object.__getattribute__(self, attr)
+        except AttributeError:
+            err_msg = (
+                f"'{attr}' is not a valid operation, you can convert this "
+                "into a pandas data frame by calling '.DataFrame()' or a "
+                "polars data frame by calling '.PolarsDataFrame()'"
+            )
+            raise AttributeError(err_msg) from None
```

Comment:
@edublancas 
Shouldn't we use `UsageError` here?

Reply:
AttributeError makes sense here to keep compatibility with the existing Python error